app/controllers/vote_accounts_controller.rb

```diff
@@ -12,22 +12,39 @@ def show
                                            .where("created_at BETWEEN ? AND ?", time_from, time_to)
                                            .order(id: :desc)
                                            .limit(60)
+
+    @explorer_stake_accounts = ExplorerStakeAccountQuery.new(
+      vote_account: va_params[:vote_account],
+      withdrawer: va_params[:withdrawer],
+      staker: va_params[:staker],
+      stake_pubkey: va_params[:stake_pubkey], 
+      network: va_params[:network]
+    ).call(page: va_params[:page], per: va_params[:per_page])
+
+    @explorer_stake_accounts_total = @explorer_stake_accounts[:total_count].to_i
+
+    @explorer_stake_accounts = @explorer_stake_accounts[:explorer_stake_accounts]
+    @stake_accounts = StakeAccount.where(stake_pubkey: @explorer_stake_accounts.pluck(:stake_pubkey))
```

Comment:
It's pretty same as in the app/controllers/explorer_stake_accounts_controller.rb.

Maybe we can wrap this into method like `set_explorer_stake_accounts` and put this into a concern?